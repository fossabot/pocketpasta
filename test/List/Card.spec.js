import { shallowMount, createLocalVue, RouterLinkStub } from '@vue/test-utils';
import Component from '@/components/List/Card';
import Vuex from 'vuex';
import BootstrapVue from 'bootstrap-vue';

const localVue = createLocalVue();

localVue.use(Vuex);
localVue.use(BootstrapVue);

describe('Setting', () => {
  const factory = () =>
    shallowMount(Component, {
      localVue,
      stubs: {
        NuxtLink: RouterLinkStub,
      },
    });

  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
