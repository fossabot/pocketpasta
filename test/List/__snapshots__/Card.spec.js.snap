// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting renders properly 1`] = `
"<b-card-stub bodytag=\\"div\\" title=\\"\\" titletag=\\"h4\\" subtitle=\\"\\" subtitletag=\\"h6\\" subtitletextvariant=\\"muted\\" headertag=\\"div\\" footertag=\\"div\\" tag=\\"div\\" no-body=\\"\\" class=\\"overflow-hidden rounded-0\\"><div class=\\"row no-gutters\\"><!----> <div class=\\"col\\"><b-card-body-stub bodytag=\\"div\\" title=\\"\\" titletag=\\"h4\\" subtitle=\\"\\" subtitletag=\\"h6\\" subtitletextvariant=\\"muted\\"><header><b-card-title-stub title=\\"\\" titletag=\\"h4\\">Recipe: undefined</b-card-title-stub></header> <b-card-text-stub texttag=\\"p\\">
          
        </b-card-text-stub></b-card-body-stub></div></div></b-card-stub>"
`;
